src/index.js

```diff
@@ -0,0 +1,67 @@
+import './style.css';
+// eslint-disable-next-line
+import { NewItemList, printDynamic, eachList, deelete
+} from './modules/Add&Remove.js';
+
+import { noCompleted } from './modules/interactive.js';
+// HTML Management
+const container = document.querySelector('.myListContainer');
+container.innerHTML = '<div class="headlineContainer"><h1>Today\'s To-Do\'s</h1><img src="https://img.icons8.com/ios-glyphs/30/000000/refresh--v1.png" class="myRefresher"></div>';
+const list = document.createElement('ol');
+list.classList.add('list');
+const input = document.createElement('div');
+input.classList.add('input');
+container.appendChild(input);
+container.appendChild(list);
+list.insertAdjacentHTML('afterend', '<button type="button" id="clearAllBtn">Clear all completed</button>');
+
+// This code below is for future improvements.
+document.querySelector('.myRefresher').addEventListener('click', () => {
+  // For future improvements and testing only console.table(eachList);
+});
+
+input.innerHTML = `
+<input type="text" id="toDo" placeholder="Add to your list..." required>
+`;
+
+const clearAllBtn = document.getElementById('clearAllBtn');
+const refreshIcon = document.createElement('img');
+refreshIcon.classList.add('mySendIcon');
+refreshIcon.src = 'https://img.icons8.com/sf-regular/48/000000/return.png';
+input.appendChild(refreshIcon);
+
+const addNewList = () => {
+  const toDo = document.getElementById('toDo').value;
+  const eachItemList = new NewItemList(toDo, false, (eachList.length + 1));
+  // Push into the array
```

Comment:
Please apply similar suggestions to line 3 and so on in the next lines that repeat the situation.

Reply:
Solved it! Thanks. 💯